python-lib/dku_stress_test_center/stress_test_center.py

```diff
@@ -31,7 +38,19 @@ def __init__(self, shift: Shift, features: list):
         super(FeaturePerturbationTest, self).__init__(shift)
         self.features = features
 
-    def _check(self, df: pd.DataFrame):
+    def check_relevance(self, preprocessing: dict):
+        if type(self.shift) is Scaling:
+            if self.shift.scaling_factor == 1:
```

Comment:
This seems a bit nitty-gritty for me, but there is no harm in having it

Reply:
Yep it is overkill/quite a stupid check 😅 But it is pretty much free.